essentials-framework/Essentials Devices Common/Essentials Devices Common/VideoCodec/ZoomRoom/ResponseObjects.cs

```diff
@@ -1510,6 +1510,8 @@ public static List<Meeting> GetGenericMeetingsFromBookingResult(List<BookingsLis
 
 				meeting.Privacy = b.IsPrivate ? eMeetingPrivacy.Private : eMeetingPrivacy.Public;
 
+				meeting.Dialable = meeting.Id != "0";

```

Comment:
Does this need to change if it's in the Zoom ResponseObject classes?  Can't we still use this equation to determine if a Zoom meeting is joinable?

I think the Cisco meetings are determined entirely by time and this should have no bearing on them?

Reply:
It does, because the `Dialable` join in the Join Map was previously using this same method to determine the value sent over the bridge, and is now using the `meeting.Dialable` property. If the Zoom class isn't setting it, then that join won't go high on the bridge, and SIMPL program relying on that value won't 

Cisco meetings can also now be scheduled without any Call information

Reply:
OH, Silly me, you're adding to the Zoom class, not removing it 🤦‍♂️